pdns/common_startup.cc

```diff
@@ -203,6 +203,8 @@ void declareArguments()
   ::arg().set("xfr-max-received-mbytes", "Maximum number of megabytes received from an incoming XFR")="100";
 
   ::arg().set("tcp-fast-open", "Enable TCP Fast Open support on the listening sockets, using the supplied numerical value as the queue size")="0";
+
+  ::arg().set("rng", "Specify random number generator to use. Valid values are auto,sodium,openssl,getrandom,arc4random,urandom,kiss.")="auto";
```

Comment:
Maybe not mention `kiss` here to protect people who don't read the docs but only --help (or the annotated pdns.conf)?